packages/SwingSet/src/liveslots/collectionManager.js

```diff
@@ -200,6 +201,24 @@ export function makeCollectionManager(
     const dbKeyPrefix = `vc.${collectionID}.`;
     let currentGenerationNumber = 0;
 
+    const keyLabel = `invalid key type for collection ${q(label)}`;
+    const valueLabel = `invalid value type for collection ${q(label)}`;
```

Comment:
These names could be better.  They are not labels, they are error messages that incorporate labels.

Reply:
Good point. Fixed.